lib/router.js

```diff
@@ -7,6 +7,7 @@
 
 var debug = require('debug')('Routr:router');
 var pathToRegexp = require('path-to-regexp');
+var queryString = require('query-string');
```

Comment:
I chose query-string since it is what fluxible-router is using and we can replace that functionality in fluxible-router with this since it will now be returned. If they are drastically different, then it could be a BC break to fluxible-router if we update fluxible-router to use the output of routr.